x/marker/client/cli/tx.go

```diff
@@ -370,3 +381,41 @@ func GetCmdWithdrawCoins() *cobra.Command {
 	flags.AddTxFlagsToCmd(cmd)
 	return cmd
 }
+
+// Transfer handles a message to send coins from one account to another
+func GetNewTransferCmd() *cobra.Command {
+	cmd := &cobra.Command{
+
+		Use:     "transfer [from] [to] [coins]",
+		Aliases: []string{"t"},
+		Short:   "Transfer coins from on account to another",
+		Args:    cobra.ExactArgs(3),
+		RunE: func(cmd *cobra.Command, args []string) error {
+			clientCtx, err := client.GetClientTxContext(cmd)
+			if err != nil {
+				return err
+			}
+			from, err := sdk.AccAddressFromBech32(args[0])
+			if err != nil {
+				return sdkErrors.Wrapf(err, "invalid from address %s", args[0])
+			}
+			to, err := sdk.AccAddressFromBech32(args[1])
+			if err != nil {
+				return sdkErrors.Wrapf(err, "invalid recipient address %s", args[1])
+			}
+			coins, err := sdk.ParseCoinsNormalized(args[2])
+			if err != nil {
+				return sdkErrors.Wrapf(sdkErrors.ErrInvalidCoins, "invalid coin %s", args[2])
+			}
+			if len(coins) != 1 {
+				return fmt.Errorf("invalid coin %s", args[2])
```

Comment:
This might need to be wrapped like above.
```suggestion
				return sdkErrors.Wrapf(sdkErrors.ErrInvalidCoins, "invalid coin %s", args[2])
```
I don't really know how much of a difference it makes, though.

Reply:
Yeah, I'm not sure either.  I always thought that wrapping with the sdk errors, should be used more on the blockchain side of things after messages are broadcasted.  I will change it to be consistent though.